lib/hex_web/email/sender.ex

```diff
@@ -0,0 +1,24 @@
+defmodule HexWeb.Email.Sender do
```

Comment:
sorry for not bringing this up earlier. with the addition of this module we have these email-related modules:

```
lib/hex_web/email/sender.ex
lib/hex_web/mail.ex
web/mailer.ex
web/models/email.ex
```

I'd like to propose that instead of adding this module, we would move this functionality as `HexWeb.Mailer.deliver_now_throttled` [1]. WDYT? cc @ericmj 

[1] This would be inline with how we e.g. extend `Repo.transaction` with a custom `Repo.transaction_with_isolation`.